### ISSUE TITLE
Release v1.1.22 .exe virus issue

### ISSUE THREAD

**Issue by yetisoldier** [kercre123/WirePod#14]
I am trying to upgrade to the latest version to utilize the new Chat GPT-4o features.  I've tried downloading the .exe file in two different browsers and both prevent it saying it has a virus.  Is this file contaminated or is it reading the file wrong?

**Comment by kercre123** [comment]
It's a false positive. The entire CI/CD process is done on GitHub Actions and I believe is viewable by all.

**Comment by yetisoldier** [comment]
![image](https://github.com/kercre123/WirePod/assets/152986698/655e5fac-c6be-426a-821b-7ee6828d031b)

**Comment by kercre123** [comment]
Yeah, you'll have to allow the threat then redownload it. I submitted it to Microsoft as an incorrect detection, so hopefully it will be resolved in a future Windows Defender definitions update. That should maybe also get rid of the smartscreen warning.

**Comment by heng1978** [comment]
Totalne nieporozumienie
